Sources/JWTKit/AppleIdentityToken.swift

```diff
@@ -0,0 +1,38 @@
+public struct AppleIdentityToken: JWTPayload {
+    enum CodingKeys: String, CodingKey {
+        case iss, aud, exp, iat, sub, nonce, email
+        case emailVerified = "email_verified"
+    }
+
+    /// The issuer-registered claim key, which has the value https://appleid.apple.com.
+    public let iss: IssuerClaim
```

Comment:
I thin it would be nice to use more readable variable names here, like https://github.com/vapor/jwt-kit/blob/master/Sources/JWTKit/JWK.swift#L139 does. Especially if we already need to declare `CodingKeys` for the underscore.

Reply:
This is kinda convention though for how people do it.

Reply:
Also, just thinking along the lines of people who aren't super familiar with JWT, the docs says you look at the 'iss' claim, for example, and they might not automatically equate that to the issuer.

Reply:
I see your point. Given that the `JWK` type takes the approach of doing full readable names though I think we should be consistent. 

I'm also imagining someone doing code review on something using JWTKit and being confused when they see:

```swift
token.hd.contains(stuff)
```

My first thought would be high-definition. I'd need to dig into the code to see what `hd` means. 

For someone implementing this, that's already looked at the docs and already knows what `hd` means, this wouldn't be problem. Typing `.hd` would probably yield `.hostedDomains` in the auto complete anyway.